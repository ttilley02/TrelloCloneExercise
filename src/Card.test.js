import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card.js';

describe('<Card />', () => {
    it('renders without crashing', () => {
        // Create a DOM element to render the component into
        const div = document.createElement('div');

        // Render the component
        // If something is wrong it will fail here
        ReactDOM.render(<Card title='blahblah' content='ham' />, div);

        // Clean up
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders without crashing', () => {
      
        const tree = renderer.create(<Card title='blahblah' content='ham' />).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});