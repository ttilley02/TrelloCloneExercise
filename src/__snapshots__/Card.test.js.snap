// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders without crashing 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    blahblah
  </h3>
  <p>
    ham
  </p>
</div>
`;
